indexd/index/drivers/alchemy.py

```diff
@@ -633,6 +635,71 @@ def add(self,
 
             return record.did, record.rev, record.baseid
 
+    def add_blank_record(self, did=None, uploader=None, baseid=None):
```

Comment:
`uploader` should not be optional for this flow, and `baseid` should not be provided